### PR TITLE
add the basic UI to the game and author classes

### PR DESCRIPTION
**Pull request by jdbs9514**
The following options should be available:
 - List of games
 - List all authors (e.g. 'Stephen King')
 - Add a game

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -6,16 +6,18 @@
...
   def initialize
-    # @items = []
+    @items = []
     # @genres = []
     @authors = []
     # @labels = []
     # @sources = []
 
-    @items = LoadData.load_items || []
+    # @items = LoadData.load_items || []
```
Comment (CindyDorantes) [inline app.rb:20]: - [ ] Kindly, modify the line that should be commented, so the app restores the Items information from the JSON file when started, and @items variable is not empty

File: classes/game.rb [hunk 2]
```diff
@@ -25,4 +27,45 @@ def two_years_old?
...
+    print 'Game title: '
+    title = gets.chomp.to_s
+
+    print 'When was it published? [yyyy-mm-dd]: '
+    publish_date = gets.chomp.to_s
+
+    print 'Is it a multiplayer game? [Y/N]: '
+    multiplayer = true if gets.chomp.to_s.downcase == 'y'
+
+    print 'when is the last played at? [yyy-mm-dd]: '
```
Comment (CindyDorantes) [inline classes/game.rb:61]: - [ ] Kindly correct the statement on how the user should provide the date, there is a 'y' missing

**Review by CindyDorantes — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

# Hi @jdbs9514 , 👋🏻👋🏻 

Good job so far!🔥🔥🔥
I really appreciate your huge efforts, there are just some details you need to work on to go to the next step but you are almost there! so kindly make the necessary changes so we can proceed with your project.💯💯✨✨✅✅✅



## Highlights ✨
- Methods implemented working correctly ✅
- Good PR description ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: data/genres.json [hunk 3]
```diff
@@ -3,14 +3,14 @@
     "id": 448,
     "name": "Alternative",
     "items_ids": [
-      506
+
```
Comment (tiagomarin) [inline data/genres.json:6]: Please put make sure this information is in this file, and stays there when the program loads and closes

File: data/genres.json [hunk 4]
```diff
@@ -3,14 +3,14 @@
...
     ]
   },
   {
     "id": 830,
     "name": "Pop",
     "items_ids": [
-      707
+
```
Comment (tiagomarin) [inline data/genres.json:13]: Please make sure this information is in this file, and stays there when the program loads and closes

File: data/items.json [hunk 5]
```diff
@@ -1,70 +1,3 @@
...
-    "title": "Motomami",
-    "genre_id": 830,
-    "author_id": "",
-    "source_id": "",
-    "label_id": "",
-    "publish_date": "2022-03-18",
-    "archived": false,
-    "on_spotify": true
-  }
+  null
```
Comment (tiagomarin) [inline data/items.json:2]: Please make sure that ALL the information in this file is preserved, and that it stays there when the program loads and closes

File: data/labels.json [hunk 6]
```diff
@@ -4,15 +4,15 @@
     "title": "Indy",
     "color": "orange",
     "items_ids": [
-      141
+
```
Comment (tiagomarin) [inline data/labels.json:7]: Please make sure this information is in this file, and stays there when the program loads and closes

File: data/labels.json [hunk 7]
```diff
@@ -4,15 +4,15 @@
...
     ]
   },
   {
     "id": 417,
     "title": "Romance",
     "color": "blue",
     "items_ids": [
-      817
+
```
Comment (tiagomarin) [inline data/labels.json:15]: Please make sure this information is in this file, and stays there when the program loads and closes

File: data/sources.json [hunk 8]
```diff
@@ -3,8 +3,7 @@
     "id": 430,
     "name": "Online shop",
     "items_ids": [
-      309,
-      255
+
```
Comment (tiagomarin) [inline data/sources.json:6]: Please make sure this information is in this file, and stays there when the program loads and closes

**Review by tiagomarin — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ 

# Hi @jdbs9514 , 👋🏻👋🏻 

Great job so far!🔥🔥🔥

Besides the changes requested by @CindyDorantes , 


please also add the necessary code to accomplish this tasks:

- [ ] All data should be preserved by saving collections in .json files.

- [ ] Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
  - games table (add all properties and associations from the parent Item class as table columns)
  - authors table

Then we'll be ready to merge all pieces together! 🔥



## Highlights ✨
- Code work as expected ✅✅✅
- Great commits 🔥🔥

## Required Changes ♻️
_Check the comments below the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by tiagomarin — APPROVED** [review]
# Pull request approved

### Hi @jdbs9514    ,

This stage of the project is complete! 
There is nothing else to say other than... it's time to merge it.

## Congratulations!



## Highlights 

- [x] Descriptive Pull request
- [x] Good use of Gitflow
- [x] Commits are meaningful
- [x] Follow best practices

**Cheers and Happy coding**

Also ,feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**